src/types.ts

```diff
@@ -1,4 +1,5 @@
 import { TransformOptions as EsBuildTransformOptions } from "https://deno.land/x/esbuild@v0.12.24/mod.js";
+import { Context } from "https://deno.land/x/oak@v10.4.0/mod.ts";
```

Comment:
better replace this import with https://deno.land/x/oak@v10.4.0/context.ts not to pull the whole Oak code